core/src/main/scala/org/apache/spark/deploy/master/ui/MasterPage.scala

```diff
@@ -75,6 +75,7 @@ private[ui] class MasterPage(parent: MasterWebUI) extends WebUIPage("") {
 
     val workerHeaders = Seq("Worker Id", "Address", "State", "Cores", "Memory")
     val workers = state.workers.sortBy(_.id)
+    val aliveWorkers = state.workers.filter(_.state == WorkerState.ALIVE )
```

Comment:
Nit: extra space at the end.

@zhichao-li  I'm OK with this change but if the UI then changes to clarify that the metrics are only for alive workers. Otherwise it introduces a smaller new confusion.

OK with that @andrewor14